pytensor/link/jax/dispatch/math.py

```diff
@@ -0,0 +1,61 @@
+import jax.numpy as jnp
+
+from pytensor.link.jax.dispatch import jax_funcify
+from pytensor.tensor.math import Argmax, Dot, Max
+
+
+@jax_funcify.register(Dot)
+def jax_funcify_Dot(op, **kwargs):
+    def dot(x, y):
+        return jnp.dot(x, y)
+
+    return dot
+
+
+@jax_funcify.register(Max)
+def jax_funcify_Max(op, **kwargs):
```

Comment:
We should have the dispatch for the other `CAReduce`, like `All, Any`... on the same file as `Max`